jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -236,6 +236,7 @@ public void claimTask(final TaskSummary task) {
                     @Override
                     public void callback(Void nothing) {
                         view.displayNotification(constants.TaskClaimed(String.valueOf(task.getId())));
+                        selectSummaryItem(task);
                         refreshGrid();
```

Comment:
I guess refreshGrid can be skipped now, as it will refresh when closing the details?

Reply:
if remove the refreshGird, when you back to task list page, the grid do not refresh.